src/renderer/processors/passthroughMeshProcessor.ts

```diff
@@ -0,0 +1,62 @@
+import {
+  Buffer,
+  BufferArray,
+  ModelNode,
+  RenderMesh,
+  RenderNode,
+} from '~/renderer/common'
+
+export const getGLPassthroughMesh: (
+  m: ModelNode,
+  gl: WebGL2RenderingContext,
+) => RenderNode = (model, gl) => {
+  const node: RenderNode = {
+    name: model.name,
+    children: [],
+  }
+
+  if (model.transform != null) {
```

Comment:
I think this should be `!== undefined`. I'm a little surprised the linter let this one through...`transform` is annotated with `?`, which should mean a `mat4` or `undefined`. Ditto for the other checks in this function.